### ISSUE TITLE
build fails: doxygen faults on missing code block terminator

### ISSUE THREAD

**Issue by algrant-arm** [DynamoRIO/dynamorio#5314]
Build fails with doxygen 1.8.3 installed as standard package on Ubuntu 18.04. Complains about missing code block terminator in scatter_gather_emulation.dox. It appears this version of doxygen doesn't like the terminator to be indented. Fix:
```
diff --git a/api/docs/scatter_gather_emulation.dox b/api/docs/scatter_gather_emulation.dox
index 3be2ca40..0f84c1e7 100644
--- a/api/docs/scatter_gather_emulation.dox
+++ b/api/docs/scatter_gather_emulation.dox
@@ -252,7 +252,7 @@ Expansion for `vpgatherdd 0x00402039(,%xmm11,4)[4byte] %xmm13 -> %xmm12 %xmm13`
  +455  m4 @0x00007fdb2ac5d900  65 48 a1 e0 00 00 00 mov    %gs:0x000000e0[8byte] -> %rax        // Restore aflags using drreg.
                                00 00 00 00
  +466  m4 @0x00007fdb2ac5d818                       <label>
- ```
+```


 Expansion for `vpscatterdd {%k1} %xmm10 -> 0x00402039(,%xmm11,4)[4byte] %k1`
@@ -368,7 +368,7 @@ Expansion for `vpscatterdd {%k1} %xmm10 -> 0x00402039(,%xmm11,4)[4byte] %k1`
  +489  m4 @0x00007fdb2ac0cc20  65 48 a1 e0 00 00 00 mov    %gs:0x000000e0[8byte] -> %rax        // Restore aflags using drreg.
                                00 00 00 00
  +500  m4 @0x00007fdb2ac0baa0                       <label>
- ```
+```

 As shown by the above expanded scatter and gather sequences, we require scratch
 registers for the expansion. The GPR scratch registers are obtained using
```

**Comment by derekbruening** [comment]
If you could submit a PR that would be great, as you already have a local fix.  The Continuous Integration tests are Ubuntu 20 so there are no automated tests of this older doxygen version.

**Comment by joshua-warburton** [comment]
Fixed by #5330
